resources.management_groups.tf

```diff
@@ -86,6 +86,14 @@ resource "time_sleep" "after_azurerm_management_group" {
     azurerm_management_group.level_6,
   ]
 
+  # This is used when var.relaxed_management_group_subscription_association is set to true
+  resource "azurerm_management_group_association" "this" {
```

Comment:
Rather than `this`, can we use consistent naming with other resources please?

Reply:
Sure - will change